sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -181,6 +181,13 @@ public abstract partial class Response<T>
         public static implicit operator T (Azure.Response<T> response) { throw null; }
         public override string ToString() { throw null; }
     }
+    public static partial class SerializationExtensions
```

Comment:
What do folks think about calling this `ObjectSerializerExtensions`?  It just makes it a little clearer exactly what it's for if anyone sees this while browsing docs, intellisense, etc.

Reply:
If we move the extension methods to ObjectSerializer, this would be left with only BinaryData extensions. Should it be called BinaryDataExtensions?

Reply:
Should we create a single extensions class in the root namespace of azure.core so we don't create a new one next time.

Reply:
I am for a single extension type for very core things (BCL like APIs). If we every API that are not really BCLy to Azure.Core, I would put their extensions into a separate class. FWIIW, serialization is a very BCLy technology.